JDBC/DatabasesPractice/src/main/java/com/laba/solvd/database/Parsing/ParsingMain.java

```diff
@@ -0,0 +1,72 @@
+package com.laba.solvd.database.Parsing;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+
+import javax.xml.XMLConstants;
+import javax.xml.namespace.QName;
+import javax.xml.stream.XMLEventReader;
+import javax.xml.stream.XMLInputFactory;
+import javax.xml.stream.XMLStreamException;
+import javax.xml.stream.XMLStreamReader;
+import javax.xml.stream.events.StartElement;
+import javax.xml.stream.events.XMLEvent;
+import javax.xml.transform.stream.StreamSource;
+import javax.xml.validation.Schema;
+import javax.xml.validation.SchemaFactory;
+import javax.xml.validation.Validator;
+
+import org.xml.sax.SAXException;
+
+public class ParsingMain {
+
+	public static void main(String[] args) throws FileNotFoundException, XMLStreamException {

```

Comment:
we cannot have two main methods in a single Java application. You already have Main.java class